css/common.css

```diff
@@ -0,0 +1,52 @@
+:root {
+  --gray: #ededee;
+  --red: #a05941;
+  --black: #133032;
+  --white: #fff;
+  --transwhite: rgba(0, 0, 0, 0.75);
+}
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+  outline: 0;
+}
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: "Outfit", sans-serif;
+}
+.btn {
+  padding: 0.5em 1em;
+  font-size: 5vw;
+  background: var(--red);
+  color: var(--white);
+}
+h1 {
+  font-size: 6vw;
+  font-family: "Fjalla One", sans-serif;
+}
+h2 {
+  font-size: 4.5vw;
+  font-family: "Outfit", sans-serif;
+}
+input {
+  padding: 0.5em 1em;
+}
+/* ============================Media Queries========================= */
+@media screen and (min-width: 481px) {
```

Comment:
481px is a little too early to be switching to desktop layout. I'd keep the mobile layout until 600px minimum. Then change a few things for tablet (even if jsut a few things like font-sizes), and go full desktop above 1000px.